beamer/tests/contracts/test_request_manager.py

```diff
@@ -79,12 +79,11 @@ def test_claim_with_different_stakes(token, request_manager, claim_stake):
 
 def test_claim_challenge(request_manager, token, claim_stake):
     """Test challenging a claim"""
-    (requester,) = alloc_accounts(1)
+    (requester, challenger) = alloc_accounts(2)
```

Comment:
Nitpick: I believe mypy (or whatever was previously complaining) will not complain if we omit parentheses here.